src/display/annotation_layer.js

```diff
@@ -718,6 +723,72 @@ var StrikeOutAnnotationElement = (
 })();
 
 /**
+ * @class
+ * @alias FileAttachmentAnnotationElement
+ */
+var FileAttachmentAnnotationElement = (
+    function FileAttachmentAnnotationElementClosure() {
+  function FileAttachmentAnnotationElement(parameters) {
+    AnnotationElement.call(this, parameters, true);
```

Comment:
Don't you need something like https://github.com/mozilla/pdf.js/blob/master/src/display/annotation_layer.js#L304-L306 here too?